common/sgx/iohandler/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ SET(GENERIC_PRIVATE_INCLUDE_DIRS "." "${TCF_TOP_DIR}/tc/sgx/common"
 	"${TCF_TOP_DIR}/tc/sgx/trusted_worker_manager/enclave")
 SET(GENERIC_CXX_FLAGS ${DEBUG_FLAGS} "-Wall" "-fPIC" "-Wno-write-strings" "-std=c++11")
 
-SET(IOHANDLER_STATIC_NAME sgx_iohandler)
+SET(IOHANDLER_STATIC_NAME iohandler)
```

Comment:
Could you rename the static library as common_sgx_iohandler. iohandler-> common_sgx_iohandler

Reply:
Done